src/pages/Home/Home.page.spec.js

```diff
@@ -0,0 +1,32 @@
+import Home from './Home.page';
+
+const { render, screen } = require('@testing-library/react');
+
+jest.mock('../../components/VideosList/VideosList', () => {
```

Comment:
Interesting mock, for this test case it makes sense since that is the child component and will be dynamic :+1: